build.fsx

```diff
@@ -242,6 +236,19 @@ Target "BuildRelease" DoNothing
 //--------------------------------------------------------------------------------
 
 //--------------------------------------------------------------------------------
+// Filter out assemblies which can't run on Linux, Mono, .NET Core, etc...
+
+open Fake.EnvironmentHelper
+let filterPlatformSpecificAssemblies (assembly:string) =
```

Comment:
Platform-specific filter for specs inside FAKE. Allows us to run the same FAKE file for multiple platforms and skip tests that can't run due to environmental issues on a particular platform.